Userland/seq.cpp

```diff
@@ -30,12 +30,23 @@
 #include <string.h>
 #include <unistd.h>
 
+static void print_usage(FILE* stream)
+{
+    fprintf(stream, "Usage:\n"
```

Comment:
Maybe use a raw string?

Reply:
What are we trying to accomplish with the raw string?

Reply:
Then you don't need to put `\n`s at the line ends and you can have a single string literal instead of one per line. Makes editing the text a bit easier.

Reply:
Would you suggest something like this?

```
    fprintf(stream, R"(Usage:
    seq [-h|--help]
    seq LAST
    seq FIRST LAST
    seq FIRST INCREMENT LAST
)"); /* Could just put fprintf(stream, '\n') on a separate line */
```

I think it makes for odd-looking code where it is, since it wouldn't follow any indention conventions. I could pull it out into a global variable, too, which would kind of fix the indention (it might not match, but it would still look ok), but it really shouldn't be a global variable since I only use it in one place.

Anyhow, tell me which approach would make you happy, and I'll do it.

Reply:
You can use 

~~~none
fprintf(stream, 1 + R"(
Usage:
    seq [-h|--help]
    seq LAST
    seq FIRST LAST
    seq FIRST INCREMENT LAST
)");
~~~

but it's a bit of a hack, raw string literals are really not useful if you care about the newlines exactly.

Reply:
@matthewcurry, yes, what you wrote is imho easier to read and edit than many string literals. I agree that the start looks a bit weird, but

* you rarely edit the first line
* hacks like the `1 +` etc don't seem worth it

Just my personal opinion though, and doesn't matter all that much :)

Reply:
I used a raw string as a global, which seems idiomatic with other uses within the project.